src/booleans.js

```diff
@@ -1,62 +1,82 @@
 function negate(a) {
-  // your code here
-};
+  return !a;
+}
 
 function both(a, b) {
-  // your code here
-};
+  return a && b;
+}
 
 function either(a, b) {
-  // your code here
-};
+  return a || b;
+}
 
 function none(a, b) {
-  // your code here
-};
+  return !(a || b);
+}
 
 function one(a, b) {
-  // your code here
-};
+  if (a && b) {
+    return false;
+  }
+  if (a || b) {
+    return true;
+  }
+  return false;
+}
 
 function truthiness(a) {
-  // your code here
-};
+  return !!a;
+//  if(a){
+//    return true
+//   } else {
+//     return false;
+//  };
+}
 
 function isEqual(a, b) {
-  // your code here
-};
+  return a === b;
+}
 
 function isGreaterThan(a, b) {
-  // your code here
-};
+  return a > b;
+}
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
-};
+  return a <= b;
+}
 
 function isOdd(a) {
-  // your code here
-};
+  if (a % 2 === 0) {
+    return false;
+  }
+  return true;
+}
 
 function isEven(a) {
-  // your code here
-};
+  return (a % 2 === 0);
+  // if (a % 2 === 0) {
+  //   return true;
+  // }
+  // return false;
+}
 
 function isSquare(a) {
-  // your code here
-};
+  return Number.isInteger(Math.sqrt(a))
+}
+//I don't really understand this one - google was my friend
 
 function startsWith(char, string) {
-  // your code here
-};
+  return string.startsWith(char)
+}
 
 function containsVowels(string) {
-  // your code here
-};
+  let pattern= /[aeiou]/i;
+  return pattern.test(string);
```

Comment:
Could condense this onto one line:
```suggestion
  return /[aeiou]/i.test(string);
```